partiql-logical-planner/src/lower.rs

```diff
@@ -1978,6 +1978,7 @@ fn lit_to_value(lit: &Lit) -> Result<Value, AstTransformError> {
     Ok(val)
 }
 
+// TODO
```

Comment:
This will be moved/removed in a future PR as part of this `dev-ion-doc` feature branch.